engine/src/main/java/org/terasology/persistence/serializers/ProtobufSerializer.java

```diff
@@ -30,38 +30,88 @@
 import java.io.InputStream;
 import java.io.OutputStream;
 
-// TODO: Document
 public class ProtobufSerializer {
+	/**
+	 * Converts the object into an array of bytes.
+	 * @see writeBytes(T, TypeHandler<T>, OutputStream) writes the serialized bytes into an OutputStream instead of a byte array
```

Comment:
This isn't quite right for the tag syntax. See reference docs: https://docs.oracle.com/javase/7/docs/technotes/tools/windows/javadoc.html#see

Mainly you just need a `#` in front of the method name and it looks like generics don't matter, so just plain `TypeHandler`

Rely on your IDE here - for me I get angry red text and some less angry highlighting of the generics brackets

Reply:
I don't have that on my IDE unfortunately